docs/_partners/integration-quality.md

```diff
@@ -0,0 +1,126 @@
+
+It can also highlight that certain triggers or actions are proving challenging to set up in a Zap or are erroring when run. Perhaps your trigger sample doesn’t return custom fields the user is looking to map in the Zap or the user gets confused by unclear input fields while setting up their action; these are two of many reasons why users abandon Zap setup. 
+
+### What we have learned
+
+Zap Activation Rates at the individual trigger and action level are a great leading indicator of performance and usability. Don’t forget about the authentication step in your integration too! If users can't authenticate or get their Zaps enabled and activated, they stand little chance of ongoing success.
+
+### Best Practices
+
+Give users the best chance to successfully activate their Zaps by making the integration as familiar and easy-to-use as possible:
+
+* Follow our [Integration Review Guidelines](https://platform.zapier.com/partners/integration-review-guidelines) which provide detailed user experience best practices for building your integration.
+* Create [Zap Templates](https://platform.zapier.com/partners/zap-templates) for popular use cases to simplify the setup process and decrease room for user error.
+* Head to the Zap Editor to create and test Zaps using your own integration. This gives you firsthand experience on how users interact with the integration. 
+* Keep your platform and Zapier integration parallel to maintain a consistent experience for mutual users.
+    * Object and field names referenced in the integration should parallel names in the platform. For example, don’t name your trigger “New Lead” if they are referred to as “Contacts” in your platform. This makes it hard for users to find the functionality they want.
+    * Provide all available input fields in the platform in the integration step, including custom fields. 
+    * Match the field types in the platform to the integration. For example, if a field is a static dropdown in the platform, implement it as the same type within the integration.
+
+
+## Bugs
+
+As users encounter bugs or think of new features they’d like within the integration, some reach out to the Zapier support team. Those requests are logged in Zapier’s issue tracker, which you can see from the Bug & Feature Requests page of the integration’s developer platform. Here are a couple things to note about bugs:
+
+1. _When the number of open bugs goes above zero, it doesn’t really matter how many bugs you have._ The important datapoint is how many of your users are affected overall and what percentage of the app’s overall monthly active users that impacts.
+2. _Your past history on closing bugs doesn’t influence your current health score._ Running track in college doesn’t mean a darn thing to your doctor thirty years later if you roll in with high cholesterol because of your penchant for McDonalds and KFC. Same with bugs and integration health.
```

Comment:
Okay I lol'd at this sentence, so good 🔥

Reply:
Ha, this is thanks to Alan! I love it too 🔥